Composer/packages/server/src/server.ts

```diff
@@ -136,13 +127,21 @@ function launchLanguageServer(socket: rpc.IWebSocket) {
   server.start();
 }
 
-attachLSPServer(wss, server, `${BASEURL}/lg-language-server`, webSocket => {
-  // launch language server when the web socket is opened
-  if (webSocket.readyState === webSocket.OPEN) {
-    launchLanguageServer(webSocket);
-  } else {
-    webSocket.on('open', () => {
+configAppInsightsAsNeeded().then(() => {
```

Comment:
@a-b-r-o-w-n  does this refactor meet your requirement on how app insights get configured?

Reply:
I was thinking we could add that in a separate pull request.

Reply:
ping @hailiu2586